NF.csv

```diff
@@ -377,6 +377,10 @@ cNF98.4c,A primary cutaneous neurofibroma cell line; not broadly available. See
 Prss56Cre;R26mT,"Prss56Cre;R26mT is a transgenic non-human animal model for Neurofibromatosis type 1, wherein the Nf1 gene is specifically inactivated in BC cells and derivatives thereof.","","",FALSE,https://patents.google.com/patent/EP3420811A1/en,modelSystemName,,"","",Prss56Cre;R26mT,Class,Biosample,Model_System,Mouse_Model,"","",""
 Nf1-OPG,"","","",FALSE,"",modelSystemName,,"","",Nf1-OPG,Class,Biosample,Model_System,Mouse_Model,"","",""
 Nf1-OPG-Arg816,"","","",FALSE,"",modelSystemName,,"","",Nf1-OPG-Arg816,Class,Biosample,Model_System,Mouse_Model,"","",""
+Nf2-/- Schwann cell line,"A Nf2-/- mouse schwann cell line","","",FALSE,"",modelSystemName,,"","",Nf2-/-Schwann,Class,Biosample,Model_System,Cell_Line_Model,"","",""
+hTERT arachnoid,"A human arachnoid cell line that has been used in drug screening (suitable control for hTERT NF2 arachnoid)","","",FALSE,"",modelSystemName,,"","",hTERT_arachnoid,Class,Biosample,Model_System,Cell_Line_Model,"","",""
+hTERT NF2 arachnoid,"A NF2 knock-out human arachnoid cell line developed with CRISPR that has been used in drug screening","","",FALSE,"",modelSystemName,,"","",hTERT_NF2_arachnoid,Class,Biosample,Model_System,Cell_Line_Model,"","",""
+hTERT meningioma,"A human meningioma cell line that has been used in drug screening","","",FALSE,"",modelSystemName,,"","",hTERT_meningioma_arachnoid,Class,Biosample,Model_System,Cell_Line_Model,"","",""
```

Comment:
probably https://web.expasy.org/cellosaurus/CVCL_1959 or https://web.expasy.org/cellosaurus/CVCL_JK01 (this lab has used this cell line in the past)

but also possibly
https://web.expasy.org/cellosaurus/CVCL_M426
https://web.expasy.org/cellosaurus/CVCL_M427
https://web.expasy.org/cellosaurus/CVCL_M428